libs/blocks/chart/list.js

```diff
@@ -0,0 +1,69 @@
+import { propertyNameCI, propertyValueCI } from './utils.js';
+
+export function listChartData(json) {
+  const data = [];
+
+  if (json[':type'] !== 'multi-sheet') {
+    console.warn('List chart error: Add table sheet');
+    return data;
+  }
+
+  const tableKey = propertyNameCI(json, 'table');
+
+  if (tableKey) {
+    json[tableKey].data?.forEach((column) => {
+      data.push({
+        title: propertyValueCI(column, 'title'),
+        list: json[propertyValueCI(column, 'sheet')]?.data,
+        type: propertyValueCI(column, 'type'),
+      });
+    });
+  } else {
+    json[':names'].forEach((sheet) => {
```

Comment:
FYI there is a character limit with using json[':names']

Reply:
Good to know. If an author has a long sheet name, they'll have to create a table instead